content/blog/2018/10/2018-10-16-custom-war-packager.adoc

```diff
@@ -103,8 +103,8 @@ plugins:
       commit: 18d78f305a4526af9cdf3a7b68eb9caf97c7cfbc
   # etc.
 systemProperties:
-    jenkins.model.Jenkins.slaveAgentPort: "9000"
-    jenkins.model.Jenkins.slaveAgentPortEnforce: "true"
+    jenkins.model.Jenkins.agentPort: "9000"
+    jenkins.model.Jenkins.agentPortEnforce: "true"
```

Comment:
See the [system properties](https://www.jenkins.io/doc/book/managing/system-properties/#jenkins-model-jenkins-slaveagentportenforce) to confirm that these can't be changed.

```suggestion
    jenkins.model.Jenkins.slaveAgentPort: "9000"
    jenkins.model.Jenkins.slaveAgentPortEnforce: "true"
```

Reply:
@MarkEWaite I understand it can't be done in this commit, but can't we add `agent*` system properties? Is this an issue that should be filed?